tools/kagami/src/genesis.rs

```diff
@@ -5,19 +5,22 @@ use clap::Subcommand;
 use crate::{Outcome, RunArgs};
 
 mod generate;
+mod hash;
 mod sign;
 
 #[derive(Debug, Clone, Subcommand)]
 pub enum Args {
     Sign(sign::Args),
     Generate(generate::Args),
+    Hash(hash::Args),
```

Comment:
```suggestion
    Generate(generate::Args),
```

could we generate both genesis block and it's hash with a single command? hash could be printed to `stdout`

Reply:
Introduced `kagami genesis prepare` command which converts `genesis.json` to `genesis.scale` and prints genesis hash to stdout